### ISSUE TITLE
Move bugdb_resolves to a static file

### ISSUE THREAD

**Issue by philip** [Sobak/bugsnet-redesign-mockup#3]
This is a suggestion to move bugdb_resolves entries out of the database and into a static file. This will make it possible for most folks to update these entries. For example, if this were static, I suspect "Try a snapshot (PHP 5.4)" would have been removed/replaced long ago by someone.

Related: eventually discuss assigning most of these to webonly as the generated bug emails contain far too many quick fix links.

**Comment by Sobak** [comment]
Hi Philip!

That's a great idea, I think, those messages are really outdated sometimes.

It seems to be out of scope for a redesign so I'll probably change it in master and then merge it to `redesign` branch which is already in progress.

**Comment by philip** [comment]
True, guess I find it so much easier to add and discuss ideas via github versus mailing lists :)

But fwiw, this lists those today:
https://bugs.php.net/admin/?action=list_responses

**Comment by Sobak** [comment]
Yup, I know that subpage, very helpful. Will take care of that soon :)
